etl/db.py

```diff
@@ -56,3 +59,73 @@ def open_db() -> Generator[DBUtils, None, None]:
             cursor.close()
         if connection:
             connection.close()
+
+
+def get_dataset_id(db_conn: MySQLdb.Connection, dataset_name: str) -> Any:
```

Comment:
I'm not sure if the `db` module is the right place for these functions. Maybe the `db_utils` makes more sense?

Reply:
I thought I should place them in the same module where function `get_connection` lives. 

I am not super sure I understand the difference between these two modules, though (`db` and `db_utils`). I discarded `db_utils` because I saw it was dedicated to the object `DBUtils`.